dll/win32/comctl32/button.c

```diff
@@ -1898,15 +1898,15 @@ static void CB_ThemedPaint(HTHEME theme, const BUTTON_INFO *infoPtr, HDC hDC, Bu
 {
     static const int cb_states[3][5] =
     {
-        { CBS_UNCHECKEDNORMAL, CBS_UNCHECKEDHOT, CBS_UNCHECKEDPRESSED, CBS_UNCHECKEDDISABLED, CBS_UNCHECKEDNORMAL },
```

Comment:
If this references data in another file it would be nice to have a comment with what it should be synchronized.

Reply:
I think is from around [wine 3.2](https://source.winehq.org/git/wine.git/blob/refs/tags/wine-3.2:/dlls/comctl32/button.c#l1290)
In [current git] it has moved to [get_draw_state](https://source.winehq.org/git/wine.git/blob/HEAD:/dlls/comctl32/button.c#l372)

Reply:
README.WINE is wrong. It was synced to wine 3.3 [see here](https://github.com/reactos/reactos/commit/b3fb8555bf9abcc04eeac31c7153d33e2f027fc0)
So this PR fixes syncing bug. I will correct REAME.WINE.